### PR TITLE
Attribute unit tests

### PR DESCRIPTION
**Pull request by 0nko**
Addresses #95.

Moved the `<p>` parsing to `AztecTagHandler` and used the existing `AztecBlockSpan` parsing logic for it.

There is one test that's commented out because list item attribute preservation is not working yet.

### CODE REVIEW

File: aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt [hunk 1]
```diff
@@ -0,0 +1,291 @@
...
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(input, output)
+    }
+
+    @Test
+    @Throws(Exception::class)
+    fun nestedAttributes() {
+        val input = NESTED
+        val expected = NESTED_REVERSED
```
Comment (theck13) [inline aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt]: Why is the order of nested attributes reversed after parsing?

File: aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+    @Throws(Exception::class)
+    fun mixedAttributes() {
+        val input = MIXED + NESTED
+        val expected = MIXED + NESTED_REVERSED
+        editText.fromHtml(input)
+        val output = editText.toHtml()
+        Assert.assertEquals(expected, output)
+    }
+
+    //TODO: After fixing list item attribute preservation, uncomment this
```
Comment (theck13) [inline aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt]: Is there an issue for list item attributes?  I saw [this](https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/98), but I'm not sure if it's related.

File: aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt (hunk 1 above)
Comment (0nko) [inline aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt]: This is a known behavior; the order of nested elements is always reversed when parsed from visual to HTML and back. I thought there existed an issue for this, but I don't see it anywhere..

File: aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt (hunk 2 above)
Comment (0nko) [inline aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt]: Yeah, it's that one. When we were merging the attributes PR with @khaykov's work on lists there was some issues with list items so we'll need to go back to it.

File: aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt (hunk 1 above)
Comment (theck13) [inline aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt]: Sounds good.  I just wanted to make sure we were tracking it.  I saw the new issue, https://github.com/wordpress-mobile/WordPress-Aztec-Android/issues/118.

File: aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt (hunk 2 above)
Comment (theck13) [inline aztec/src/test/kotlin/org/wordpress/aztec/AttributeTest.kt]: That works.  I just wanted to make sure we were tracking it.

**Comment by theck13** [comment]
:shipit:

**Pull request merged**